modules/afsocket/afsocket-source.c

```diff
@@ -35,6 +35,19 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 
+#ifdef SYSLOG_NG_HAVE_LINUX_SOCK_DIAG_H
+#include <linux/sock_diag.h>
+
+#ifdef SO_MEMINFO
+
+#define SYSLOG_NG_ENABLE_PACKET_STATS 1
+
+static const glong PACKET_STATS_TIMER_MSECS = 1000;
```

Comment:
Isn't this a very short time if there are a lot of UDP sources or SO_REUSEPORT UDP sources?
Should we make this configurable?

Reply:
I wouldn't want that. maybe 1/sec is a bit too often, but I think we should decide the frequency ourselves. The overhead of the getsockopt in question is not a lot, it's basically a single system call overhead, of which we do 10s of thousands per second, anyway.

Reply:
Okay. Thank you.